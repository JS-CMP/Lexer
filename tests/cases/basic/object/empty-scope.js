var a = "Hello";
if (a == "Hello") {
} else if (a == "Hello") {
} else {
}
for (var v = 0; v < 10; v++) {
}
while (false) {
}
do {
} while (false);
/* switch (a) {
}*/
