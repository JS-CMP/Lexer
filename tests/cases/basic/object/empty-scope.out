#include "types/JsAny.hpp"
#include "global/global.hpp"
#include "types/objects/Types.hpp"

int main() {JS::Any a=JS::Any("Hello");if (a==JS::Any("Hello")){}else if (a==JS::Any("Hello")){}else {};for (JS::Any v=JS::Any(0);v<JS::Any(10);v++){};while (JS::Any(false)){};do {}while (JS::Any(false));switch (a){};}
